kubernetes/internal/create-weblogic-domain.sh

```diff
@@ -584,6 +605,30 @@ function createDomain {
 
 }
 
+#
+# Deploy voyager/HAProxy load balancer
+#
+function setupVoyagerLoadBalancer {
+  # deploy voyager ingress controller
+  kubectl create namespace voyager
```

Comment:
Should this namespace be configurable?

Reply:
This is the namespace of Voyager Controller, not Voyager Ingress resource. We only need one Voyager Controller. I think it's fine to hardcoded it.